### PR TITLE
add resize icon to all studio apps including applab, playlab, etc.

### PR DESCRIPTION
**Pull request by davidsbailey**
screencast (40MB for < 30 seconds, sorry)
http://www.gfycat.com/HomelyOpulentAfghanhound

### CODE REVIEW

File: apps/style/applab/style.scss [hunk 1]
```diff
@@ -41,6 +41,9 @@ div#visualization.with_padding {
   html[dir='rtl'] div#visualizationResizeBar {
     right: $width;
   }
+  #visualizationResizeIcon {
```
Comment (cpirich) [inline apps/style/applab/style.scss]: to ensure that the applab specific styles override the common ones, this file should use `div#visualizationResizeIcon` (both here and the rtl version below)

File: apps/src/templates/page.html.ejs [hunk 2]
```diff
@@ -78,6 +78,7 @@
 </div>
 
 <div id="visualizationResizeBar">
+  <i id="visualizationResizeIcon" class="fa fa-ellipsis-v"></i>
```
Comment (cpirich) [inline apps/src/templates/page.html.ejs]: Surprisingly to me to see the `<i>` tag here. Long debate here on whether or not it makes any sense: http://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span

File: apps/src/applab/applab.js [hunk 3]
```diff
@@ -250,6 +250,11 @@ function adjustAppSizeStyles(container) {
               childRules[k].style.cssText = "right: " +
                   Applab.appWidth * scale + "px;";
               changedChildRules++;
+            } else if (childRules[k].selectorText === "#visualizationResizeIcon") {
```
Comment (cpirich) [inline apps/src/applab/applab.js]: Use `div#visualizationResizeIcon` here (see comment below)

Comment (cpirich) [inline apps/src/applab/applab.js]: also, the `changedChildRules < 8` above should now be `changedChildRules < 9`. Feel free to move the number to a constant :smile:

**Comment by cpirich** [comment]
Looks good! All comments complete. I'd be fine with you merging after making the changes that I suggested.

However, I wonder if it might be possible to place the text/icon directly on the visualization resize bar div and not have to create another element that needs to have responsive styles added, modified, etc. I can't tell if you tried that and determined that we needed to create another element specifically to render this glyph.

**Comment by sfilman** [comment]
@davidsbailey can we do the secondary gray color from the stylesheet instead of the primary charcoal?
image

File: apps/src/templates/page.html.ejs (hunk 2 above)
Comment (cpirich) [inline apps/src/templates/page.html.ejs]: One more question: typically, for proper design encapsulation, we only reference classes in apps that are defined by CSS in the apps directory. I can't find `fa-ellipsis-v` - is that defined somewhere else?

Comment (davidsbailey) [inline apps/src/templates/page.html.ejs]: fa is "font awesome". here is what they have to say on the topic: "You can place Font Awesome icons just about anywhere using the CSS Prefix fa and the icon's name. Font Awesome is designed to be used with inline elements (we like the `<i>` tag for brevity, but using a `<span>` is more semantically correct)." http://fortawesome.github.io/Font-Awesome/examples/

Comment (davidsbailey) [inline apps/src/templates/page.html.ejs]: here is the particular icon in question: http://fortawesome.github.io/Font-Awesome/icon/ellipsis-v/

**Comment by davidsbailey** [comment]
Here's how it looks with #c6cacd 
screen shot 2015-06-04 at 11 47 44 am
screen shot 2015-06-04 at 11 46 40 am

**Pull request merged**